docs/2.7.0/docs/canton/usermanual/example_monitoring_setup.rst

```diff
@@ -221,19 +213,18 @@ See `Grafana Metric Monitoring`_ for instructions of how to log into Grafana and
 Dependencies
 ~~~~~~~~~~~~
 
-There are startup dependencies between the docker containers, for example the domain needs to be running before the particpant, and in turn, the datbase needs to run before the domain.
+There are startup dependencies between the Docker containers. For example, the domain needs to be running before the participant, and the database needs to run before the domain.
 
-The `yaml` anchor `x-logging` enabled GELF container logging and is duplicated across the containers where we want to capture logging output. One thing to note is that the host address used is that of the host machine, not a network address (on OSX).
+The `yaml` anchor `x-logging` enabled GELF container logging and is duplicated across the containers where you want to capture logging output. Note that the host address is the host machine, not a network address (on OSX).
```

Comment:
@carrielaben-da @hrischuk-da I had trouble making sense of the sentence beginning with "The 'yaml' anchor". Hoping someone can help me edit it?

Reply:
Highly technical gibberish.   Suggest accepting it

Reply:
Thanks, @hrischuk-da! I wasn't sure whether there was or wasn't a verb in the sentence. I'll leave it be.